test/tools/ossfuzz/protoToYul.h

```diff
@@ -344,6 +344,8 @@ class ProtoConverter
 	static unsigned constexpr s_modOutputParams = 5;
 	/// Hard-coded identifier for a Yul object's data block
 	static auto constexpr s_dataIdentifier = "datablock";
+	/// Upper bound on memory writes
+	static unsigned constexpr s_maxMemory = 1024;
```

Comment:
@bshastry wanna change that? We could get this merged soon

Reply:
The bound suggested by @hrkrshnn would not work with the current state of the Yul interpreter because it does not properly interpret memory accesses to high locations such as 2**32

Reply:
It could easily do so, though, couldn't it? Resp. the bound in `EVMInstructionInterpreter::accessMemory` is 
`numeric_limits<size_t>::max() - 0xffff`... on most platforms `numeric_limits<size_t>::max()` should be 2**64... or do you mean a different limitation in there?

Reply:
I was wrong. Thanks for correcting @ekpyron . I pushed an update with a few more changes.